### PR TITLE
change bytesize topic

### PR DESCRIPTION
**Pull request by FranBonath**
@netlify /events/2024/bytesize_resolve_merge_conflics

### CODE REVIEW

File: src/content/events/2024/bytesize_resolve_merge_conflics.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+type: talk
+startDate: '2024-05-14'
+startTime: '13:00+00:00'
+endDate: '2024-05-14'
+endTime: '13:30+00:00'
+locations:
+  - links: https://kth-se.zoom.us/j/68390542812
+---
+
+What Kübra ([@kubranarci](https://github.com/kubranarci)) live, as she is reloving merge conflics resulting from the newest tools release in the [pull request for nf-core/variantbenchmarking](https://github.com/nf-core/variantbenchmarking/pull/37)!
```
Comment (mashehu) [inline src/content/events/2024/bytesize_resolve_merge_conflics.md:13]:
```suggestion
Whatch Kübra ([@kubranarci](https://github.com/kubranarci)) live, as she is resolving merge conflicts and the changes from the automated PRresulting from the newest tools release in the [pull request for nf-core/variantbenchmarking](https://github.com/nf-core/variantbenchmarking/pull/37)!
```

**Review by kubranarci — APPROVED** [review]

**Comment by kubranarci** [comment]
Hello @FranBonath @mashehu , I committed to do this but not very sure about the format :D Is it possible to direct me to similar bytesize talk?

**Review by maxulysse — APPROVED** [review]

File: src/content/events/2024/bytesize_Tamper_Monkey.md [hunk 2]
```diff
@@ -1,24 +0,0 @@
----
-title: 'Bytesize: Tamper Monkey'
-subtitle: Phil Ewels, Seqera
-type: talk
-startDate: '2024-05-14'
```
Comment (ewels) [inline src/content/events/2024/bytesize_Tamper_Monkey.md:5]: Just change the date rather than deleting?

**Review by ewels — CHANGES_REQUESTED** [review]

File: src/content/events/2024/bytesize_Tamper_Monkey.md (hunk 2 above)
Comment (FranBonath) [reply src/content/events/2024/bytesize_Tamper_Monkey.md:5]: I kept a copy of it on my computer, was just waiting for you to give me a new date.

**Comment by FranBonath** [comment]
Merging this by bypassing branch protection, so we have it up on the website asap.

**Pull request merged**